Makefile.in

```diff
@@ -380,11 +381,16 @@ $(CURL_VERSION)/.firefox: $(firefox_libs) $(CURL_VERSION).tar.xz $(CURL_VERSION)
 	  fi; \
 	  add_cflags="-I$(nss_install_dir)/../public/nss"; \
 	  add_cflags+=" -I$(nss_install_dir)/include/nspr"; \
+	  if test "$(host_cpu)" = "arm"; then \
+	    add_libs="-L$(nss_install_dir)/lib -l:libgcm-aes-arm32-neon_c_lib.a"; \
```

Comment:
I think this should go in the curl patch file, look for the patch to `libcurl.pc.in` where these flags are set.